dashboards/jupyterhub.jsonnet

```diff
@@ -342,45 +381,45 @@ dashboard.new(
 ).addPanel(
   userNodes, { x: 0, y: 0 } + standardDims
 ).addPanel(
-  clusterUtilization, { x: 12, y: 0 } + standardDims
+  clusterUtilization, { x: standardDims.w, y: 0 } + standardDims
```

Comment:
We can pass just `{}` (or just a `y` or `x`) to these - `deploy.py` autolayouts now.